integration/spark/shared/src/main/java/io/openlineage/spark/agent/lifecycle/SparkOpenLineageExtensionVisitorWrapper.java

```diff
@@ -285,6 +285,11 @@ private static void loadProviderToAvailableClassloaders(List<ClassLoader> classl
             if (!(ex.getCause() instanceof LinkageError)) {
               log.error("Failed to load OpenLineageExtensionProvider class", ex.getCause());
             }
+          } catch (RuntimeException ex) {
+            // On Java 17, if add opens is not used, the defineClass method will throw an
+            // InaccessibleObjectException
+            // Any workaround is TODO, but we should not crash here
+            log.debug("{}: Failed to load OpenLineageExtensionProvider class ", cl, ex);
```

Comment:
We can add an extra condition and debug only if `ex.getClass().getCanonicalName().endsWith("InaccessibleObjectException")`

Otherwise, this should be still `error`

Reply:
I think error in `SparkOpenLineageExtensionVisitorWrapper` should not error whole Spark integration.

Reply:
> I think error in SparkOpenLineageExtensionVisitorWrapper should not error whole Spark integration.

This's not erroring whole Spark integration. It's just logging error message. 
Perhaps one should prevent logging the stacktrace, or lower logging level to `warn`.

But I am not sure if we should silence the message, as we know there is some problem.